prov/rxm/src/rxm_cq.c

```diff
@@ -1014,8 +1006,8 @@ int rxm_cq_open(struct fid_domain *domain, struct fi_cq_attr *attr,
 		goto err1;
 
 	*cq_fid = &util_cq->cq_fid;
-	/* Override util_cq_fi_ops */
-	(*cq_fid)->fid.ops = &rxm_cq_fi_ops;
+	/* Override util_cq_fi_ops::close */
+	(*cq_fid)->fid.ops->close = rxm_cq_close;
```

Comment:
This doesn't work.  The entire ops structure points to a single set of function calls.  You need to override 'ops', rather than a single function pointer.

Reply:
Yes - that would work

Reply:
Done